ranger/gui/ui.py

```diff
@@ -272,6 +272,9 @@ def handle_input(self):
                 else:
                     if not self.fm.input_is_blocked():
                         self.handle_key(key)
+            elif key == -1 and not os.isatty(0):
```

Comment:
Could you use `sys.stdin.fileno()` rather than the hard coded `0`?

Reply:
Good idea, fixed.